ProjectShop/ProductApp/models.py

```diff
@@ -57,13 +75,21 @@ class Product(models.Model):
 
     subcategories = models.ManyToManyField(ProductSubcategory)
     categories = models.ManyToManyField(ProductCategory)
-    tags = models.ManyToManyField(Tag)
+    tags = models.ManyToManyField(Tag, related_name='products')
 
     name = models.CharField(max_length=100, null=False, blank=False)
-    price = models.DecimalField(validators=[MinValueValidator(0)], decimal_places=2, null=False, blank=False)
+    price = models.DecimalField(validators=[MinValueValidator(0)], decimal_places=2, max_digits=9,
+                                null=False, blank=False)
     description = models.TextField(max_length=5000, null=False, blank=False)
     stock_quantity = models.IntegerField(validators=[MinValueValidator(0)], null=False, blank=False)
 
+    def __str__(self):
+        return self.name
+
+    @property
+    def short_description(self):
+        return truncatechars(self.description, 120)
```

Comment:
```suggestion
        return truncatewords(self.description, 20)
```
Возможно так лучше будет

Reply:
done